modules/rank-eval/src/test/java/org/opensearch/index/rankeval/DiscountedCumulativeGainTests.java

```diff
@@ -91,20 +91,20 @@ public void testDCGAt() {
         DiscountedCumulativeGain dcg = new DiscountedCumulativeGain();
         assertEquals(EXPECTED_DCG, dcg.evaluate("id", hits, rated).metricScore(), DELTA);
 
-        /**
-         * Check with normalization: to get the maximal possible dcg, sort documents by
-         * relevance in descending order
-         *
-         * rank | relevance | 2^(relevance) - 1 | log_2(rank + 1) | (2^(relevance) - 1) / log_2(rank + 1)
-         * ---------------------------------------------------------------------------------------
-         * 1 | 3 | 7.0 | 1.0  | 7.0
-         * 2 | 3 | 7.0 | 1.5849625007211563 | 4.416508275000202
-         * 3 | 2 | 3.0 | 2.0  | 1.5
-         * 4 | 2 | 3.0 | 2.321928094887362 | 1.2920296742201793
-         * 5 | 1 | 1.0 | 2.584962500721156  | 0.38685280723454163
-         * 6 | 0 | 0.0 | 2.807354922057604  | 0.0
-         *
-         * idcg = 14.595390756454922 (sum of last column)
+        /*
+          Check with normalization: to get the maximal possible dcg, sort documents by
+          relevance in descending order
+          <p>
```

Comment:
No need for `<p>` here (internal method comments)

Reply:
fixed in new commit

Reply:
I see it still there?

Reply:
i missed some. re patching.